Project4/styles.css

```diff
@@ -0,0 +1,151 @@
+@import url(https://fonts.googleapis.com/css?family=Montserrat);
+
+$nav-height: 60px;
+h1 h2 h3 {
+  color: #ffc400;
+}
+body {
+  text-align:center;
+  margin:0px;
+  padding:0px;
+  font-family:'Montserrat', 'Oswald', 'Poppins', 'PT Sans',
+  'Titillium Web', 'Nunito', 'PT Sans Narrow',
+  'Rubik', 'Arimo', 'Arvo', 'Cabin', 'Fjalla One',
+  'Abel', 'Kanit', san-serif;
+  color:#001d1f;
+}
+p {
+  text-align:left;
+  margin:0px;
+  padding:20px;
+  font-family:'Montserrat', 'Oswald', 'Poppins', 'PT Sans',
```

Comment:
You declared the same fonts on the `body` element, so you shouldn't need to repeat them here.